examples/12. Advanced Action.md

```diff
@@ -0,0 +1,55 @@
+FORMAT: 1A
+
+# Advanced Action API
+A resource action is – in fact – a state transition. This API example demonstrates that an action - state transition - to another resource.
```

Comment:
The last sentence does not make sense. Please re-read it loud & then improve it.